### PR TITLE
Separate highlighting for quickfix selected line - different than 'Search'

### PR DESCRIPTION
**Pull request by anishsane**
Currently, the quickfix window selected entry is shown with the same highlighting as `Search` group.
Because of this, it's difficult to search for something within the quick fix window.
Plus I would rather like the ability to disable the highlighting for that line.

Issue filed: https://github.com/vim/vim/issues/1080

### CODE REVIEW

File: src/vim.h [hunk 1]
```diff
@@ -1433,6 +1433,7 @@ typedef enum
     , HLF_CUC	    /* 'cursurcolumn' */
     , HLF_CUL	    /* 'cursurline' */
     , HLF_MC	    /* 'colorcolumn' */
+    , HLF_QFC	    /* 'quickfix window current line' */
```
Comment (dpelle) [inline src/vim.h]: The single quotes are meant to be around the Vim options  (such as 'colorcolumn', 'cursorline', etc.), not around the full comment. Here it's not a vim option, so don't put the single quote.

Comment (anishsane) [reply src/vim.h]: Thanks. Updated the comment; removed the quotes.

**Comment by wincent** [comment]
[Important observation about this](https://github.com/vim/vim/issues/1080#issuecomment-306957503):

> This exists in Neovim but it is named `QuickFixLine`. I don't think "Current" qualifier is needed. Would be nice to use the same name.

**Comment by anishsane** [comment]
Now the automated check is failing with "Coverage decreased (-0.001%) to 77.42%"

1. Does it really matter for 0.001% decrease?
2. I just changed one enum name and a string value since my earlier commit. Why would that change anything in coverage?

I am asking because the latest commit is showing "failed at testing stage".

**Comment by chrisbra** [comment]
such failures are usually ignored and are no obstacle for merging this function

**Comment by chrisbra** [comment]
but a test would be good

**Pull request closed without merging**